src/tools/decomp.py

```diff
@@ -1,35 +1,81 @@
-#  Copyright (C) 2012,2013,2016
```

Comment:
please run ```autopep8``` tool on this file, it is quite difficult to read the current formatting.

Reply:
@MrTheodor, OK will do the check!